src/pages/storage/forms/DuplicateVolumeForm.tsx

```diff
@@ -136,7 +143,7 @@ const DuplicateVolumeForm: FC<Props> = ({ volume, close }) => {
       const targetProject =
         values.project === formik.initialValues.project ? "" : values.project;
 
-      duplicateStorageVolume(payload, values.pool, targetProject)
+      duplicateStorageVolume(payload, values.pool, targetProject, member)
```

Comment:
There is a comment in the api function
```
// Including project and target params if they did not change from source configs breaks the API call.
// Therefore, we only include them if they are different from the source configs, that's why both project and target are optional inputs
```
Can you validate the statement in the comment is still correct?

If yes, I think we should set the member here to empty string or null if the value is equal to the initial value. Kind of like the `targetProject` calculation above.